spyder/plugins/variableexplorer/widgets/tests/test_arrayeditor.py

```diff
@@ -239,6 +239,7 @@ def test_arrayeditor_edit_2d_array(qtbot):
     assert np.sum(diff_arr != dlg.get_value()) == 2
 
 
+@pytest.mark.slow
```

Comment:
@ccordoba12 I think I wanted to use 

`@pytest.mark.first` but this also worked 😆